webapp/channels/src/components/user_settings/modal/user_settings_modal.scss

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+.UserSettingsModal {
```

Comment:
I see that `.settings-modal` is used in some other places, so we need to override its CSS rules now, but are there any rules that can be removed from `_settings.scss` to avoid having to use `!important` and such here?

I'm looking for things like `.adminModeBadge` where there's conflicting styles defined in this file and in `_settings.scss` even though that class is only used in this component